### PR TITLE
Add reading JSON from file.

### PR DESCRIPTION
**Pull request by andrey-star**
* Add a FileService which reads contents of a file with an http.get request
* Subscribe to the returned Observable in the Environment component

_Testing Observables requires research. Will add it later not to block other people._

### CODE REVIEW

File: step-release-vis/src/app/services/environment.ts [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
 export class EnvironmentService {
 
-  constructor() {
+  constructor(private fileService: FileService) {
   }
 
   // xs: 0-100, ys: timestamps
-  getPolygons(jsonFile: string): Polygon[] {
-    const environments = this.readJson(jsonFile);
+  getPolygons(jsonFile: string): Observable<Polygon[]> {
```
Comment (stofee) [inline step-release-vis/src/app/services/environment.ts:18]: test for this?

Comment (andrey-star) [reply step-release-vis/src/app/services/environment.ts:18]: Testing Observables requires research. I thought I'd add it later not to block other people.

Comment (stofee) [reply step-release-vis/src/app/services/environment.ts:18]: // TODO(...) ... :)

Comment (andrey-star) [reply step-release-vis/src/app/services/environment.ts:19]: My bad, added the TODO.

**Pull request merged**